GraphicsView/src/CGAL_Qt5/CMakeLists.txt

```diff
@@ -84,7 +84,14 @@ endif(CGAL_HEADER_ONLY)
 collect_cgal_library( CGAL_Qt5 "${additional_files}")
 
 if(CGAL_HEADER_ONLY)
-   set(CGAL_QT5_QTMOCCMAKE_FILES ${QTMOCCMAKE_FILES} CACHE INTERNAL "CGAL qt5 qtmoc cmake files")
+  FILE(GLOB files "../../include/CGAL/Qt/*")
+  install(FILES ${files} DESTINATION "${CGAL_INSTALL_CMAKE_DIR}/include/CGAL/Qt/")
```

Comment:
You want to install CGAL *headers* in the the directory for CGAL CMake files (`${CGAL_INSTALL_CMAKE_DIR}`)?!